<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteTab</class>
 <widget class="QWidget" name="RouteTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Routes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QSpinBox" name="mIdSb"/>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="mDistanceSb">
     <property name="suffix">
      <string> km</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mIdLabel">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="mDailyRepeatsLabel">
     <property name="text">
      <string>Daily Repeats</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="mDistanceLabel">
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QSpinBox" name="mDailyRepeatsSb">
     <property name="suffix">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QTableWidget" name="mRouteTableWidget"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="mNewRouteBtn">
     <property name="text">
      <string>New Route</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="mDelRouteBtn">
     <property name="text">
      <string>Delete Route</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="mPointsEdit"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="mPointsLabel">
     <property name="text">
      <string>Points</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
